data/human/weapons.txt

```diff
@@ -1372,3 +1560,199 @@ effect "bullet impact"
+	
+outfit "Shell Hold"
+	category "Ammunition"
+	"outfit space" 20
+	"cost" 3000
+	"shell capacity" 500
+	"ammo" "HC Shell"
+	"mass" 0.85
+	
+outfit "HC Shell"
+	category "Ammunition"
+	cost 500
+	thumbnail "outfit/meteor"
+	"mass" .2
+	"shell capacity" -1
+	
+outfit "Heavy Barrage Bullet"
+	weapon
+		safe
+		phasing
+		sprite "projectile/ion bolt"
+			rewind
+		sound "explosion small"
+		"hit effect" "barrage hit"
+		lifetime 100
+		"random lifetime" 20
+		"hull damage" 179
+		"shield damage" 454
+		piercing .05
+		"blast radius" 15
+		"hit force" 10
+		
+effect "barrage hit"
+	sprite "effect/explosion/medium"
+		"no repeat"
+		"frame rate" 10
+		"random start frame"
+	"lifetime" 28
+	sound "explosion small"
+	"random angle" 360
+	"random spin" 3
+	"velocity" .1
+	drag 50
+	"acceleration" .001
+	
+effect "barrage fire"
+	sprite "effect/explosion/medium"
+		"no repeat"
+		"frame rate" 30
+	"lifetime" 5
+	"velocity" 12
+	"random angle" 360
+	"lifetime" 60
+	"random spin" 60
+	"random frame rate" 60
+
+effect "small barrage fire"
+	sprite "effect/explosion/tiny"
+		"no repeat"
+		"frame rate" 30
+	"lifetime" 5
+	"velocity" 5
+	"random angle" 360
+	"lifetime" 60
+	"random spin" 60
+	"random frame rate" 60
+	
+outfit "Enceladus Cannon"
+	category "Turrets"
+	cost 770000
+	thumbnail "outfit/blaster turret"
+	"mass" 70
+	"outfit space" -60
+	drag 3
+	"weapon capacity" -56
+	"turret mounts" -1
+	"required crew" 3
+	"shell capacity" 200
+	weapon
+		sprite "projectile/ion bult" 3
+		"hardpoint sprite" "hardpoint/blaster turret"
+		"hardpoint offset" 9.
+		sound "detainer"
+		"ammo" "HC Shell"
+		"fire effect" "small barrage fire"
+		"hit effect" "explosion/small"
+		"inaccuracy" 1
+		velocity 22
+		"lifetime" 60
+		"reload" 60
+		"burst count" 8
+		"burst reload" 30
+		"relative firing energy" .004
+		"firing heat" 90
+		"shield damage" 187
+		"turret turn" .44
+		"hull damage" 569
+		"blast radius" 1
+		"piercing" .2
+		parallel
+		cluster
+		
+	description "This large turret fires explosive shells at a moderate rate, and can punch through light and medium armour. Against the heaviest ships, this turret doesn't do too much damage, but combined with its successor, the Tethys Cannon, the amount of firepower this turret throws at the target is terrifying." 
+	
+outfit "Light Flak Turret"
+	category "Turrets"
+	cost 360000
+	thumbnail "outfit/proton turret"
+	drag .5
+	mass 66
+	"outfit space" -6
+	"weapon capacity" -5
+	"turret mounts" -1
+	"required crew" 4
+	weapon
+		sprite "projectile/ion bult"
+		"hardpoint sprite" "hardpoint/proton turret"
+		"hardpoint offset" 14.
+		"hit effect" "proton impact" 3
+		"turret turn" 2
+		"submunition" "Heavy Proton Split" 7
+		"inaccuracy" 20
+		"velocity" 35
+		"lifetime" 8
+		"reload" 40
+		"relative firing energy" .001
+		"firing heat" 60
+		"blast radius" 1
+		"split range" 100
+		"shield damage" -4200
+		"hull damage" -2400
```

Comment:
```suggestion
		"shield damage" 4200
		"hull damage" 2400
```

Reply:
I added the negative so it would only show the damage of each individual projectile. If I didn't add the negative on the main weapon, it would show that damage otherwise. The weapon technically does not do 4k damage anyway. It probably deals about 20% of an interceptor's shields per shot.